pkg/testing/tools/estools/elasticsearch.go

```diff
@@ -392,42 +428,11 @@ func CheckForErrorsInLogsWithContext(ctx context.Context, client elastictranspor
 						},
 					},
 				},
+				"must_not": excludeStatements,
 			},
 		},
 	}
 
-	if len(excludeStrings) > 0 {
-		excludeStatements := []map[string]interface{}{}
-		for _, ex := range excludeStrings {
-			excludeStatements = append(excludeStatements, map[string]interface{}{
-				"match_phrase": map[string]interface{}{
-					"message": ex,
-				},
-			})
-		}
-		queryRaw = map[string]interface{}{
-			"query": map[string]interface{}{
-				"bool": map[string]interface{}{
-					"must": []map[string]interface{}{
-						{
-							"match": map[string]interface{}{
-								"log.level": "error",
-							},
-						},
-						{
-							"term": map[string]interface{}{
-								"data_stream.namespace": map[string]interface{}{
-									"value": namespace,
```

Comment:
[Blocker]
Have you forgotten to add this part of the query, the one that selects the data stream and the error logs?

Reply:
ah, I missed the other part because it was not changed.